internal/transport/http/handler.go

```diff
@@ -246,6 +246,7 @@ func groupV1Routes(e *echo.Group, h *Handler) {
 	playlistGroup.POST("/:id", h.AddVodToPlaylist, auth.GuardMiddleware, auth.GetUserMiddleware, auth.UserRoleMiddleware(utils.EditorRole))
 	playlistGroup.DELETE("/:id/vod", h.DeleteVodFromPlaylist, auth.GuardMiddleware, auth.GetUserMiddleware, auth.UserRoleMiddleware(utils.EditorRole))
 	playlistGroup.DELETE("/:id", h.DeletePlaylist, auth.GuardMiddleware, auth.GetUserMiddleware, auth.UserRoleMiddleware(utils.EditorRole))
+	playlistGroup.PUT("/:id/delay", h.SetVodDelayOnPlaylist, auth.GuardMiddleware, auth.GetUserMiddleware, auth.UserRoleMiddleware(utils.EditorRole))
```

Comment:
```suggestion
	playlistGroup.PUT("/:id/delay", h.SetVodDelayOnPlaylistMultistream, auth.GuardMiddleware, auth.GetUserMiddleware, auth.UserRoleMiddleware(utils.EditorRole))
```
Adding `Multistream` to the function just so it's more clear what the function is. The function name will need to be updated in the other places as well.

Reply:
Updated!